### ISSUE TITLE
BrightStartDB assembly references Newtonsoft.Json 7.0.1 but the NuGet packages still references NewtonSoft.Json 6.0.8

### ISSUE THREAD

**Issue by jvdonk** [BrightstarDB/BrightstarDB#249]
Which results in a runtime error:

![image](https://cloud.githubusercontent.com/assets/1164988/11142043/d4dcd7e8-89ea-11e5-85a7-4eed7fe73478.png)

and a build warning:

````
warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.
````

When updating the project(s) to NewtonSoft.Json 7.0.1 everything works fine.

**Comment by kal** [comment]
It turns out that up until now we have been grandfathering the Newtonsoft.JSON dependency in via our dependency on dotNetRDF. But it seems we have diverged in terms of minimum version number required so I've updated our nuspec templates (commit f4edd41)  to explicitly specify our dependency on 7.0.1.

Unfortunately on the machine I have available right now I cannot test a full package build, but hopefully once our build server has churned through I'll have a package I can test to ensure that this doesn't break the NuGet dependency manager :)

**Comment by jvdonk** [comment]
Thank you for your efforts. While this was just a heads up issue(at least for me),  your quick and accurate responses are very comforting when exploring new software.

**Comment by kal** [comment]
Fixed for 1.12 release
